### PR TITLE
Release 0.9.0

### PR DESCRIPTION
**Pull request by vfusco**

### CODE REVIEW

**Review by mpernambuco — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,8 +5,25 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+## [0.9.0] - 2023-08-14
+### Fixed
+- Fix lua-check warnings
+- Fix peer dependencies warning
+- Replace `@types/commander` with `commaner`
+- Remove and update npm packages for security reason
+
+### Added
+- Added `dep` and `depclean` target
+
```
Comment (edubart) [inline CHANGELOG.md:18]: I think is important to always mention which emulator release this solidity-step release is expected to work with.

Currently, the main branch is incompatible with the emulator main branch, so we cannot release it yet.

**Review by edubart — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,8 +5,25 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+## [0.9.0] - 2023-08-14
+### Fixed
+- Fix lua-check warnings
+- Fix peer dependencies warning
+- Replace `@types/commander` with `commaner`
+- Remove and update npm packages for security reason
+
+### Added
+- Added `dep` and `depclean` target
```
Comment (stephenctw) [inline CHANGELOG.md:17]: `dep` and `depclean` are no longer supported, since we don't install `npm` modules anymore.

Comment (stephenctw) [reply CHANGELOG.md:17]: `commander` is no longer required.

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,8 +5,25 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+
 ### Changed
 - Updated license/copyright notice in all source code
+- Removed npm-scripts-info package
```
Comment (stephenctw) [inline CHANGELOG.md:21]: + Changed `step` function interface and parameters to be more generic types.

**Comment by vfusco** [comment]
Cherry picked into #46

**Pull request closed without merging**